Sources/CountDownLatch.swift

```diff
@@ -14,12 +14,12 @@ import Foundation
 import XCTest
 
 /// CountDown latch to be used for asserts and expectations
-class CountDownLatch {
-    private let initialCount: Int32
-    private var currentCount: Int32
-    private let waitSemaphore = DispatchSemaphore(value: 0)
+open class CountDownLatch {
+    public let initialCount: Int32
+    public var currentCount: Int32
+    public let waitSemaphore = DispatchSemaphore(value: 0)
```

Comment:
I think the intention here was to keep these properties private to the class, and only expose them through the getters. Can we revert this change?